packages/form-js-viewer/src/Form.js

```diff
@@ -158,7 +177,7 @@ export default class Form {
   }
 
   /**
-   * @returns { { [x: string]: string[] } }
+   * @returns {Errors}

```

Comment:
Is this a breaking change? Or rather a refinement of the existing types? If it is a refinement, we should use it in other places, i.e. [here](https://github.com/bpmn-io/form-js/pull/85/files#diff-53d9b079c7db646b79736d7ab07cd7d3108049b780d14637f41c188a4f6ffc03R20), too.

Reply:
We simply didn't use the existing `Errors` type here.